evalml/automl/engine/dask_engine.py

```diff
@@ -1,9 +1,11 @@
 import joblib
+from dask.distributed import Client
```

Comment:
Ah good call @freddyaboulton . Yep I see dask distributed [is a requirement for featuretools](https://github.com/alteryx/featuretools/blob/main/requirements.txt#L7).

As long as `distributed` is getting installed as part of evalml, we can choose whichever representation we want. I'd list it in our requirements if we need a different version from featuretools, but otherwise IMO its fine to omit it in the evalml reqs.

This raises questions for me about why `distributed` is a hard requirement for featuretools rather than an optional one. I don't know the answer. @rwedge @gsheni @thehomebrewnerd any thoughts on that?

There's a future question here too about how hard it is to support different backends like ray--if we can do that well eventually, dask distributed and the other parallel execution frameworks don't need to be core requirements of evalml.

Reply:
This was added as a hard requirement for Featuretools because we add `n_jobs` as as argument to DFS. If a user increased `n_jobs` from 1 (default), we wanted Featuretools to keep working.

Here is the associated line when we added multiprocessing to Featuretools:
- https://github.com/alteryx/featuretools/pull/170/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552R11